docs/platform-integration/essentials/platform-feature-support.md

```diff
@@ -1,54 +1,54 @@
 ---
-title: "Xamarin.Essentials Platform & Feature Support"
-description: "Xamarin.Essentials provides a single cross-platform API that works with any iOS, Android, or UWP application that can be accessed from shared code no matter how the user interface is created."
-ms.date: 08/20/2019
+title: "Essentials platform and feature support matrix"
+description: "Learn about what features of .NET MAUI Essentials each platform supports."
+ms.date: 08/11/2021
 no-loc: ["Microsoft.Maui", "Microsoft.Maui.Essentials"]
 ---
 
-# Platform Support
+# Platform support
 
-Xamarin.Essentials supports the following platforms and operating systems:
+This article describes which features of .NET Multi-platform App UI (.NET MAUI) Essentials are supported on the following platforms:
 
-| Platform | Version |
-| --- | --- |
-| Android | 4.4 (API 19) or higher |
-| iOS |10.0 or higher |
-| Tizen | 4.0 or higher |
-| tvOS | 10.0 or higher |
-| watchOS | 4.0 or higher |
-| UWP | 10.0.16299.0 or higher |
-| macOS | 10.12.6 (Sierra) or higher |
+| Platform | Version                    |
```

Comment:
I'd put a todo comment here to revisit this table. For starters, https://docs.microsoft.com/dotnet/maui/supported-platforms lists the min platform requirements for MAUI. I don't think we need to duplicate this info now that Essentials is part of MAUI. Also, I'm not sure if we need to be calling out tvOS/watchOS anymore.

Also, it's Windows now, not UWP (no UWP support in WinUI 3).